SECURITY.md

```diff
@@ -0,0 +1,93 @@
+# Notation Security Process and Policy
+This document provides the details on the Notation security policy and details the process surrounding security handling including a how to report a security vulnerability for anything within the Notary Project organization. 
```

Comment:
Seems only Notation related repo were listed below for VA report, should we mention `Notary Project organization`? Will `notary` repo be covered?

Reply:
This is a good point. I added the other two repos that have code in them.